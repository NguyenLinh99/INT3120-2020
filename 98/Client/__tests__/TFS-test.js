import 'react-native';
import React from 'react';

import renderer from 'react-test-renderer';
import Item from '../components/TFS/TFSCItem';

it('TFS snapshot', () => {
    const snap = renderer.create(
        <Item />
    ).toJSON();
    expect(snap).toMatchSnapshot();
});