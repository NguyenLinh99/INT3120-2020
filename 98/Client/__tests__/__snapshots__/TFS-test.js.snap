// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TFS snapshot 1`] = `
<View
  accessible={true}
  focusable={true}
  isTVSelectable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
>
  <View
    style={
      Object {
        "elevation": 1.6,
        "flexDirection": "row",
        "height": 70,
        "marginLeft": 20,
        "marginRight": 20,
        "marginTop": 10,
      }
    }
  >
    <Image
      source={
        Object {
          "uri": "http://192.168.1.174:3000undefined",
        }
      }
      style={
        Object {
          "height": 50,
          "marginLeft": 20,
          "marginTop": 10,
          "width": 50,
        }
      }
    />
    <Text
      style={
        Object {
          "alignContent": "center",
          "color": "#696969",
          "fontSize": 18,
          "fontWeight": "bold",
          "justifyContent": "center",
          "marginLeft": 15,
          "marginTop": 15,
        }
      }
    />
  </View>
</View>
`;
